### PR TITLE
Remove unneeded coding from LaTeXTranslator.visit_figure()

### PR DESCRIPTION
**Pull request by jfbu**
Simple trimming of unneeded conditionals

### CODE REVIEW

File: sphinx/writers/latex.py [hunk 1]
```diff
@@ -1853,28 +1853,14 @@ def visit_figure(self, node):
...
         elif self.in_minipage:
-            if ('align' not in node.attributes or
-                    node.attributes['align'] == 'center'):
-                self.body.append('\n\\begin{center}')
-                self.context.append('\\end{center}\n')
-            else:
-                self.body.append('\n\\begin{flush%s}' % node.attributes['align'])
-                self.context.append('\\end{flush%s}\n' % node.attributes['align'])
+            self.body.append('\n\\begin{center}')
+            self.context.append('\\end{center}\n')
```
Comment (jfbu) [inline sphinx/writers/latex.py]: the align attribute has already been tested at https://github.com/sphinx-doc/sphinx/pull/4429/files#diff-81c44cde8d80608ae558b93878f9be8aR1846

File: sphinx/writers/latex.py [hunk 2]
```diff
@@ -1853,28 +1853,14 @@ def visit_figure(self, node):
...
-                align = '\n\\centering'
-                align_end = ''
-            else:
-                # TODO non vertical space for other alignments.
-                align = '\\begin{flush%s}' % node.attributes['align']
-                align_end = '\\end{flush%s}' % node.attributes['align']
-            self.body.append('\n\\begin{figure}[%s]%s\n' % (
-                self.elements['figure_align'], align))
+            self.body.append('\n\\begin{figure}[%s]\n\\centering\n' %
+                             self.elements['figure_align'])
```
Comment (jfbu) [inline sphinx/writers/latex.py]: same here, align has been tested, only remaining option is to center

File: sphinx/writers/latex.py (hunk 1 above)
Comment (jfbu) [inline sphinx/writers/latex.py:1857]: the align attribute has already been tested at https://github.com/sphinx-doc/sphinx/pull/4429/files#diff-81c44cde8d80608ae558b93878f9be8aR1846

File: sphinx/writers/latex.py (hunk 2 above)
Comment (jfbu) [inline sphinx/writers/latex.py:1860]: same here, align has been tested, only remaining option is to center

**Review by tk0miya — APPROVED** [review]

**Comment by jfbu** [comment]
thanks for review @tk0miya, merging.

**Pull request merged**